gradle.properties

```diff
@@ -16,6 +16,6 @@ kotlin.native.ignoreDisabledTargets=true
 # workaround dokka bug (need to wait for next snapshot build)
 org.jetbrains.dokka.classpath.excludePlatformDependencyFiles=true
 
-artifactVersion = 2.2.0
+artifactVersion = 2.3.0-SNAPSHOT
```

Comment:
Publication versions should not be labeled "SNAPSHOT"

Reply:
fixed